app/build.gradle

```diff
@@ -38,7 +41,10 @@ dependencies {
     implementation "com.android.support:design:${supportLibVersion}"
     implementation "com.android.support:exifinterface:${supportLibVersion}"
     implementation "com.android.support:support-v4:${supportLibVersion}"
+    implementation "com.google.android.gms:play-services-gcm:${playServicesVersion}"
```

Comment:
Why is this necessary?

Reply:
This is for the Maps getting last known location

Reply:
Can you link documentation for that? I'm pretty sure gcm is referring to Google Cloud Messaging

Reply:
Any heads up on this? Unnecessary dependencies have plagued this repo for too long.

Reply:
Yeah, the gcm part is for cloud messaging, for maps it is play-services-map. I might have miswritten it

Reply:
Great! Also check the other ones to be sure.